dataactvalidator/scripts/load_object.py

```diff
@@ -0,0 +1,55 @@
+import os
+import logging
+
+import pandas as pd
+import boto
+
+from dataactcore.config import CONFIG_BROKER
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+from dataactcore.models.domainModels import ObjectClass
+from dataactvalidator.health_check import create_app
+from dataactvalidator.scripts.loaderUtils import clean_data, insert_dataframe
+
+logger = logging.getLogger(__name__)
+
+
+def load_object_class(base_path):
+
+    """
+    This function loads Object classes into the database
+        Args:
+            base_path : directory that contains the domain values files.
+    """
+    if CONFIG_BROKER["use_aws"]:
+        s3connection = boto.s3.connect_to_region(CONFIG_BROKER['aws_region'])
+        s3bucket = s3connection.lookup(CONFIG_BROKER['sf_133_bucket'])
+        filename = s3bucket.get_key("object_class.csv").generate_url(expires_in=600)
+    else:
+        filename = os.path.join(base_path, "object_class.csv")
+    # Load object class lookup table
```

Comment:
If this is referring to the code below, there shouldn't be a space below it, only above.